### PR TITLE
Adding support for Script Metadata(Aegisub Project Garbage)

### PR DESCRIPTION
**Pull request by Zahuczky**

### CODE REVIEW

File: AssCS/AssParser.cs [hunk 1]
```diff
@@ -26,8 +26,7 @@ public static AssFile LoadAndParse(string filePath)
                     "[v4+ styles]" => ParseStyleLine,
                     "[events]" => ParseEventLine,
                     "[script info]" => ParseScriptInfoLine,
-                    "[aegisub project garbage]" => ParseMetadataLine,
-                    _ => ParseUnknownLine
+                    "[aegisub project garbage]" => ParseMetadataLine
```
Comment (9vult) [inline AssCS/AssParser.cs:29]: Please add the `_ => ParseUnknownLine` clause back, this is the default clause for the switch statement.

File: AssCS/AssParser.cs [hunk 2]
```diff
@@ -68,10 +67,17 @@ private static void ParseScriptInfoLine(string line, AssFile file)
...
-
     private static void ParseMetadataLine(string line, AssFile file)
     {
-        // TODO
+        var pair = line.Split(": ");
+        if (pair.Length >= 2)
+            file.ScriptMetadata.Add
+            (
+                pair[0].Trim(),
+                pair[1].Trim()
```
Comment (9vult) [inline AssCS/AssParser.cs:77]: I would use `string.Join()` here to account for potential colons in the metadata value (eg dummy videos)
See [here](https://github.com/9vult/Ameko/blob/23e71eee7a03df3a168045e75842176343cc9d7c/AssCS/AssEvent.cs#L55) for an example in AssEvent.

**Review by 9vult — CHANGES_REQUESTED** [review]
Overall looks pretty good 👍 Just address the couple things I pointed out and we'll be good for a merge.

File: AssCS/AssParser.cs (hunk 2 above)
Comment (Zahuczky) [reply AssCS/AssParser.cs]: Done.

**Pull request merged**